fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -357,6 +367,18 @@ private void singleRebuild(@Nonnull List<TestRecords1Proto.MySimpleRecord> recor
             }
 
             buildFuture.join();
+
+            // if a record is added to a range that has already been built, it will not be counted, otherwise,
+            // it will.
+            long additionalScans = 0;
+            if (recordsWhileBuilding != null && recordsWhileBuilding.size() > 0) {
+                additionalScans += (long)recordsWhileBuilding.size();
+            }
+            assertThat(indexBuilder.getTotalRecordsScanned(),
+                    Matchers.allOf(
+                            Matchers.greaterThanOrEqualTo((long)records.size()),
+                            Matchers.lessThanOrEqualTo((long)records.size() + additionalScans)
```

Comment:
The matchers should be statically imported.

Reply:
Will do.